### DISCUSSION TITLE
empty response from /sites endpoint

### DISCUSSION THREAD

**Discussion by guzthabbo** [amberelectric/public-api#79] [General]
Hello there, 
I need to retrieve my sites list, so I can get the required sitesId parameter to use the other API methods. At the moment I'm getting empty results from postman: 

![postman](https://user-images.githubusercontent.com/4857960/137252867-0de47aad-83b7-42e7-bd57-550268973bcf.JPG)

And the developers page (app.amber.com.au/developers):

![developer_page](https://user-images.githubusercontent.com/4857960/137252745-e45e6e76-2e8c-42c6-9d0e-31fedac15b9c.JPG)

Am I missing anything?

**Comment by madpilot** [comment]
This usually means you are such a new customer, we haven't finished receiving your information from the network, or you are an old customer with no active site, or (more rarely) something has gone wrong with your account.

Send an email to info@amber.com.au (Subject line: API isn't returning sites) with your account details and I can investigate further.
